### PR TITLE
More new features

### PR DESCRIPTION
**Pull request by Peteck**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no

### Route for getting index of pages for a specified section
This may be useful for if you have section that you want to use as an navigation.

### Route for getting multiple sections.
This may be useful for if you wan't to create an footer with multiple columns. This way you can use sections this way.
`{{ render(path('bitbag_sylius_cms_plugin_shop_section_show_codes', {'codes' : 'footer_column_1,footer_column_2,footer_column_3,footer_column_4,footer_column_5', 'template' : '@SyliusShop/_footerColumn.html.twig'})) }}`
And in the column template like this
```
{% for section in sections %}
<div class="footer-column">
    <label for="{{ section.code }}">{{ section.name }}</label>
    <input type="checkbox" id="{{ section.code }}">
    <div class="column-info">
        {{ render(path('bitbag_sylius_cms_plugin_shop_block_index_by_section_code', {'sectionCode' : section.code, 'template' : '@SyliusShop/_footerBlocks.html.twig'})) }}
    </div>
</div>
{% endfor %}
```

### CODE REVIEW

**Comment by patrick477** [comment]
Hello! Thank you for your PR. Can I ask you to fix the build?

**Comment by Peteck** [comment]
I've fixed the test, but I had to constraint the composer to Sylius v1.3 as v1.4 breaks the behat tests. Please let me know if you want me to revert this, but the tests will not pass then.

**Comment by bitbager** [comment]
That is really cool! Thank you. Could you please add a documentation reference as well?

File: src/Repository/PageRepository.php [hunk 1]
```diff
@@ -87,6 +87,21 @@ public function createShopListQueryBuilder(string $sectionCode, string $channelC
...
+            ->leftJoin('o.translations', 'translation')
+            ->innerJoin('o.sections', 'section')
+            ->where('translation.locale = :localeCode')
+            ->andWhere('section.code = :sectionCode')
+            ->andWhere('o.enabled = true')
+            ->setParameter('sectionCode', $sectionCode)
+            ->setParameter('localeCode', $localeCode)
+            ->getQuery()
+            ->getResult()
+            ;
```
Comment (bitbager) [inline src/Repository/PageRepository.php]: Could you move this a tab (4 spaces) to the left?

**Comment by Peteck** [comment]
There you go :-)

**Comment by bitbager** [comment]
Thank you again 🍻

**Pull request merged**